lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -185,9 +185,11 @@ const std::string Fp16_to_Fp8E4M3Nv = "{ \n"
 // Fp8E4M3 (x2) -> Fp16 (x2) (packed)
 const std::string Fp8E4M3Nv_to_Bf16 =
     "{                                       \n"
+    ".reg .b32 a;                            \n"
     ".reg .f16 a<2>;                         \n"
-    ".reg .bf16 b<2>;                        \n"
-    "cvt.rn.f16x2.e4m3x2 {a0, a1}, $1;       \n"
+    ".reg .b16 b<2>;                         \n"
+    "cvt.rn.f16x2.e4m3x2 a, $1;              \n"
+    "mov.b32 {a0, a1}, a;                    \n"
```

Comment:
Not sure is there a way to avoid this mov?

Reply:
it probably doesn't matter much, it can get optimized away by ptxas if possible.